openSUSE/open-build-service
# Rename component to DeleteConfirmationDialogComponent

In order to have a more descriptive name.

How to test that there isn't any regression?

On watchlist:
- Open your watchlist
- Add a project if you don't have one
- Remove that project by clicking on the (x) icon next to the project name.
- A confirmation dialog should appear, it should contain the name of the project
- Click on the "Remove" button and the project should disappear from the list

On token's users page:
- Go to _Your Profile_ > _Manage Your Tokens_
- Click on the users icon
- Add one user to the list if you don't have one
- Remove the user from the list
- A confirmation dialog should appear, it should contain the name of the user
- Click on the "Remove" button and the user should disappear from the list
